clusterDeployer.py

```diff
@@ -354,6 +361,8 @@ def create_cluster(self) -> None:
         cfg["vip_dhcp_allocation"] = False
         cfg["additional_ntp_source"] = "clock.redhat.com"
         cfg["base_dns_domain"] = "redhat.com"
+        if self._sno:
+            cfg["sno"] = True
```

Comment:
Would `cfg["sno"] = self._is_sno()` work too?

Reply:
Should work, with the above change it will be the return of the function.